PocketKit/Tests/PocketKitTests/AppBadgeTrackerTests.swift

```diff
@@ -0,0 +1,63 @@
+import XCTest
+@testable import PocketKit
+import Combine
+import SharedPocketKit
+@testable import Sync
```

Comment:
🔥 woo hoo tests!~ Not a blocker, but would like to see the `delete` case as well

Reply:
ok